src/response_code.rs

```diff
@@ -51,6 +51,7 @@ bitfield! {
 }
 
 /// Rust native representation of the TSS2 response codes as defined in the spec.
+#[allow(clippy::module_name_repetitions)]
```

Comment:
Could this be put at the crate root with `#![allow(clippy::module_name_repetitions)]` to avoi putting it in all modules?

Reply:
There might appear valid cases of it at some point, I found these two to be false-positives, but who knows